policy/approval/approve.go

```diff
@@ -57,15 +57,19 @@ type RequestReview struct {
 
 func (opts *Options) GetMethods() *common.Methods {
 	methods := opts.Methods
+	defaultBool := true
```

Comment:
Let's use a more descriptive name here and in the other instances where `defaultBool` has been added. Perhaps `defaultGithubReview`?